proprietary/tokens/src/components/ams/dialog.tokens.json

```diff
@@ -3,11 +3,11 @@
     "dialog": {
       "background-color": { "value": "{ams.color.primary-white}" },
       "border": { "value": "0" },
-      "inline-size": { "value": "calc(100% - 2 * {ams.space.grid.md})" },
-      "max-block-size": { "value": "calc(100% - 2 * {ams.space.grid.md})" },
+      "inline-size": { "value": "calc(100dvw - 2 * {ams.space.grid.md})" },
       "max-inline-size": { "value": "48rem" },
-      "form": {
+      "content": {
         "gap": { "value": "{ams.space.md}" },
+        "max-block-size": { "value": "calc(100dvh - 4 * {ams.space.grid.md})" },
```

Comment:
Pretty great browser support? Usage relative is 92% and when is anything 100%? Even 'width' is 96%... 

DVH has great benefits: https://web.dev/blog/viewport-units
Onscreen keyboard behaviour is different per browser/device: https://developer.chrome.com/blog/viewport-resize-behavior#viewport_resize_behavior and I think we should test an input in the Dialog to test how this will work out before drawing conslusions. But it's possibly another reason not to touch the default display behaviour of the dialog.

Reply:
Also 100% does not seem to work well in safari iOS.
![Simulator Screenshot - iPhone 15 - 2024-09-13 at 14 45 58](https://github.com/user-attachments/assets/d4b72eff-82c6-486a-a6f6-00ce64e43b85)

Reply:
I think the question we should answer here is: Do we want Dialog to change size when the keyboard is shown? I think it's a little odd, squashing the readable text when the keyboard appears. You'll especially notice that with longer texts.

Reply:
It does not seem to change size, opening the keyboard does not change the height of elements. The content of dialog wil continue to determine the size. Or am I missing something here? If you can show me a case where the dialog is squashed I can have a go at it.